src/Pagerfanta/Adapter/SphinxAdapter.php

```diff
@@ -0,0 +1,84 @@
+<?php
+/*
+ * This file is part of the Pagerfanta package.
+ *
+ * (c) Maikel Doezé <maikel.doeze@marqin.nl>
```

Comment:
Can you remove the copyright message?  I know that many of the files have a copyright message relating to Pablo Diez, but for simplicity, we want to move towards one copyright framework for the whole project rather than per-file statements.